hbt/config/variables.py

```diff
@@ -187,57 +191,32 @@ def add_variables(config: od.Config) -> None:
         expression="cutflow.jet1_pt",
         binning=(40, 0.0, 400.0),
         unit="GeV",
-        x_title=r"Jet 1 $p_{T}$",
+        x_title=r"Leading jet $p_{T}$",
     )
     add_variable(
         config,
         name="cf_jet1_eta",
         expression="cutflow.jet1_eta",
         binning=(40, -5.0, 5.0),
-        x_title=r"Jet 1 $\eta$",
+        x_title=r"Leading jet $\eta$",
     )
     add_variable(
         config,
         name="cf_jet1_phi",
         expression="cutflow.jet1_phi",
-        binning=(32, -3.2, 3.2),
-        x_title=r"Jet 1 $\phi$",
+        binning=(34, -3.3, 3.3),
```

Comment:
```suggestion
        binning=(66, -3.3, 3.3),
```
If you don't want to go through zero, bins with size 0.194... are not the solution, either you don't care and stay with 33 bins with size 0.2 or you go with 66 bins with size 0.1